build/packer/pmm.json

```diff
@@ -113,47 +114,28 @@
       "type": "shell",
       "inline": [
         "sudo yum -y update",
+        "sudo dnf install -y rsync jq podman",
         "sudo yum -y install epel-release",
         "sudo yum -y install ansible-core",
-        "sudo yum -y install ansible-collection-community-general ansible-collection-community-postgresql ansible-collection-ansible-posix"
+        "sudo yum -y install ansible-collection-community-general ansible-collection-community-postgresql ansible-collection-ansible-posix ansible-collection-community-docker"
       ]
     },
     {
       "type": "ansible-local",
-      "playbook_file": "ansible/pmm2/main.yml",
+      "playbook_file": "packer/ansible/pmm.yml",
       "extra_arguments": [
-        "-v",
+        "-vvv",
         "-u root",
         "--extra-vars",
-        "\"pmm2_server_repo='{{ user `pmm2_server_repo`}}' pmm_client_repo_name='{{ user `pmm_client_repo_name`}}' pmm_client_repos='{{ user `pmm_client_repos`}}'\""
+        "\"pmm2_server_repo='{{ user `pmm2_server_repo`}}' pmm_client_repo_name='{{ user `pmm_client_repo_name`}}' pmm_client_repos='{{ user `pmm_client_repos`}}' pmm_server_image_name='{{ user `pmm_server_image_name`}}'\""
```

Comment:
`mm2_server_repo='{{ user `pmm2_server_repo`}}` is no longer necessary, let's clean it up.